### PR TITLE
Issue_3: min encoder

### LINKED ISSUE #3: Implement Min encoder for KBinsDiscretizer

### PR DESCRIPTION
**Pull request by linjun9**
#### Reference Issues/PRs
issue reference: https://github.com/VishwaP98/lets-git-it-scikit-learn/issues/3
parent PR: https://github.com/VishwaP98/lets-git-it-scikit-learn/pull/1

#### What does this implement/fix? Explain your changes.
find the minimum value in each bin after KBinsDiscretizer allocates each entry to a specific bin.

### CODE REVIEW

**Comment by linjun9** [comment]
@VishwaP98 @RylandWang @shrutimistry @vishwaapatel @alalalau review please 😄 
Screen Shot 2021-03-06 at 8 54 46 PM

File: sklearn/preprocessing/tests/test_discretization.py [hunk 1]
```diff
@@ -206,14 +206,39 @@ def test_nonuniform_strategies(
     Xt = est.fit_transform(X)
     assert_array_equal(expected_5bins, Xt.ravel())
 
+def test_transform_min():
+
+
```
Comment (VishwaP98) [inline sklearn/preprocessing/tests/test_discretization.py]: remove extra line

**Comment by VishwaP98** [comment]
minor comment but otherwise looks good to me!

File: sklearn/preprocessing/tests/test_discretization.py (hunk 1 above)
Comment (linjun9) [reply sklearn/preprocessing/tests/test_discretization.py]: updated!

**Review by VishwaP98 — APPROVED** [review]

**Review by shrutimistry — APPROVED** [review]
looks good!

**Pull request merged**